modules/microshift-multus-intro.adoc

```diff
@@ -6,21 +6,49 @@
+You can use an additional network in situations where network isolation is needed, including control plane and data plane separation. You can create additional interfaces for pods to connect to that network in addition to a default. For example, you can configure an additional interface if you want pods to access a network on the host and also communicate with devices deployed to the edge. These edge devices might be on an isolated operator network or are periodically disconnected.
 
 Isolating network traffic is useful for the following performance and security reasons:
 
 Performance:: You can send traffic on two different planes to manage the amount of traffic on each plane.
 Security:: You can send sensitive traffic onto a network plane that is managed specifically for security considerations, and you can separate private data that must not be shared between tenants or customers.
 
-[id="additional-network-how-implemented"]
+[NOTE]
+====
+The {microshift-short} service deploys the Multus CNI plugin at start up, so a host restart might be required depending on when the `microshift-multus` RPM package is installed.
+====
+
+[id="microshift-additional-network-how-implemented_{context}"]
 == How additional networks are implemented
 
-All of the pods in the cluster still use the cluster-wide default network to maintain connectivity across the cluster. Every pod has an `eth0` interface that is attached to the cluster-wide pod network. You can view the interfaces for a pod by using the `oc exec -it <pod_name> \-- ip a` command. If you add additional network interfaces that use Multus CNI, they are named `net1`, `net2`, ..., `netN`.
+All of the pods in the cluster still use the cluster-wide default network to maintain connectivity across the cluster. Every pod has an `eth0` interface that is attached to the cluster-wide pod network. You can view the interfaces for a pod by using the `oc exec -it <pod_name> \-- ip a` command. If you add additional network interfaces that use the {microshift-short} Multus CNI, they are named `net1`, `net2`, ..., `netN`.
+
+[id="microshift-additional-network-how-attached-pods_{context}"]
+== How additional networks are attached to pods
+
+To attach additional network interfaces to a pod, you must create configurations that define how the interfaces are attached.
+
+* Applying a YAML manifest: You must specify each interface by using a `NetworkAttachmentDefinition` custom resource (CR). A configuration inside each of these CRs defines how that interface is created.
+* You must configure other parts of your networking, such as other CNIs, to use the {microshift-short} Multus CNI plugin. Because of individual differences in networks, no default configuration is provided.
```

Comment:
So it's Multus that calls other CNIs. Other CNIs don't need to be aware of it.
But for cluster/kubelet/crio to use Multus, CRIO needs to be configured but we provide a file for this - only thing needed for CRIO to use it, is restart CRIO (unless you have running Pods, then we want to reboot whole system)